modules/aleph/lib/AlephClient/AlephClient.class.php

```diff
@@ -135,29 +142,20 @@ public function authenticate($bor_id, $verification, $library = 'ICE53', $sub_li
       // Check if the user is blocked for each sub-library.
       $is_blocked = FALSE;
 
-      $block_codes = array(
+      $sub_libraries = array(
```

Comment:
Add a comment describing the contents of the array (eg. the element-names of the status-code and the message for the status respectivly).
You can also consider adding some sample-xml in a comment to make it clear what we're looking for.